ecli/build.rs

```diff
@@ -11,10 +11,10 @@ fn main() {
     println!("cargo:rustc-link-search=../bpf-loader/build/libbpf");
 
     println!("cargo:rustc-link-lib=static=eunomia");
-    println!("cargo:rustc-link-lib=static=stdc++");
+    println!("cargo:rustc-link-lib=dylib=stdc++");
     println!("cargo:rustc-link-lib=static=bpf");
-    println!("cargo:rustc-link-lib=static=elf");
-    println!("cargo:rustc-link-lib=static=z");
+    println!("cargo:rustc-link-lib=dylib=elf");
+    println!("cargo:rustc-link-lib=dylib=z");
```

Comment:
Is it really necessary to statically link `libstdc++`, `libelf` and `libz` ? @yunwei37
 
These three libs are generally pre-installed on most Linux distros, and static linking caused some weird things to happen while separating lib from ecli.
https://stackoverflow.com/questions/64304659/c-library-cannot-be-linked-when-main-rs-and-lib-rs-coexist

This also happened when I tried to detach ecli into lib-server-cli.

Reply:
No. I think maybe you can replace them with dyn link.